integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/CoinbaseServicesFragment.kt

```diff
@@ -47,6 +55,9 @@ class CoinbaseServicesFragment : Fragment(R.layout.fragment_coinbase_services) {
     private val viewModel by viewModels<CoinbaseServicesViewModel>()
     private var loadingDialog: FancyAlertDialog? = null
     private var currentExchangeRate: org.dash.wallet.common.data.ExchangeRate? = null
+    private val sharedViewModel: CoinbaseActivityViewModel by activityViewModels()
+
+    @Inject lateinit var analyticsService: AnalyticsService
```

Comment:
This injection doesn't look like it's needed there. The viewModel has "logEvent" method that should be used instead.
In general, avoid injections into fragments when possible (e.g. when the methods don't require context or activity)